Resources/Prototypes/Entities/Objects/Devices/hand_teleporter.yml

```diff
@@ -9,10 +9,10 @@
     layers:
     - state: icon
   - type: HandTeleporter
-  - type: ReverseEngineering # Nyano
-    difficulty: 4
-    recipes:
-      - HandTeleporter
+  #- type: ReverseEngineering # Nyano
```

Comment:
This should just be removed or if you want to comment it out then should be tagged with a Delta comment

Reply:
Done!